examples/play.py

```diff
@@ -1,8 +1,18 @@
+from pathlib import Path
+from PIL import Image
 import argparse
 import pystk
 from time import time
+import numpy as np
 from . import gui
 
+
+def action_to_numpy(action):
```

Comment:
numpy? It's it a string? To make it a bit easier to read, maybe we can make this a dict and save it as `str(dict)`?
```
def action_dict(action):
    return {k: getattr(action,k) for k in ['acceleration', 'brake', 'steer', 'fire', 'drift']}
```

Reply:
i like action_dict better, then read from it with eval()?